solidity/contracts/StaticOracle.sol

```diff
@@ -57,7 +58,7 @@ contract StaticOracle is IStaticOracle {
     uint32 _period
   ) external view override returns (uint256 _quoteAmount, address[] memory _queriedPools) {
     _queriedPools = _getQueryablePoolsForTiers(_baseToken, _quoteToken, _period);
-    _quoteAmount = _quote(_baseAmount, _baseToken, _quoteToken, _queriedPools, _period);
+    _quoteAmount = _quote(_baseAmount, _baseToken, _quoteToken, _queriedPools, _period, 0);
```

Comment:
I'm curious. Is there a reason you prefer to call `_quote` instead of calling `quoteAllAvailablePoolsWithOffsettedTimePeriod` with a `0` offset here?
I though it could make sense to call `quoteAllAvailablePoolsWithOffsettedTimePeriod` just to avoid code repetition, but maybe you are thinking of a reason to avoid doing so?

Reply:
oh, smart, i think i went for the 'less changes as possible' approach, calling that method would also require to make that one a `public` one instead of `external`, but since now there are 2 paths, it'd be possible to merge those 2 paths like that, or just make them call to the same methods. 
i'm open to codeowner's preference here!

Reply:
routing all `WithTimePeriod` through `WithOffsettedTimePeriod`
there is one that directly called `_quote`, but i think it looks cleaner like this

Reply:
can revert `quoteSpecificPoolsWithTimePeriod` into just calling `_quote` also